_posts/2017-11-20-Running-CAW-with-Singularity.md

```diff
@@ -59,13 +57,13 @@ docker build -t ${repository}/${container}:${tag} ${baseDir}/containers/${contai
 docker push ${repository}/${container}:${tag}
 ```
 
-Since Singularity can directly pull images from Docker-hub, I slightly modified my building script to pull all containers from Docker-hub to have local Singularity images.
+Since Singularity can directly pull images from DockerHub, I made the build script to all containers from DockerHub to have local Singularity image files.
```

Comment:
I don't recognize this phrase anymore, and Rickard was having problem with it yesterday too, I think it's gladly improved, it's just missing a work I think.

> Since Singularity can directly pull images from DockerHub, I made the build script to **pull** all containers from DockerHub to have local Singularity image files

Reply:
You're right, my version above makes no sense 👍 

> In order to have local image files which can be transferred by SFTP, I made the build script pull all containers from DockerHub to local Singularity images.